ballerina/http_service_endpoint.bal

```diff
@@ -110,6 +101,26 @@ public isolated class Listener {
             return <readonly & InferredListenerConfiguration> self.inferredConfig.cloneReadOnly();
         }
     }
+
+    function createInterceptors(Interceptor|Interceptor[]|CreateInterceptorsFunction? interceptorConfigValues) returns Interceptor[] {
+        Interceptor|Interceptor[]|CreateInterceptorsFunction? configValues = interceptorConfigValues;
```

Comment:
Any specific reason for creating a local variable here? Since we are not updating interceptorConfigValues, we can avoid this local variable right?

Reply:
We cannot assign values to function parameters. So had to create `configValues` and assign return value from the `CreateInterceptorsFunction` to it

Reply:
Got it 👍🏼